src/open_inwoner/pdc/views.py

```diff
@@ -217,6 +220,7 @@ def get_context_data(self, **kwargs):
         context["anchors"] = anchors
         context["related_products_start"] = 6 if product.links.exists() else 1
         context["product_links"] = product.links.order_by("pk")
+        context["readmore"] = readmore
```

Comment:
You don't really need 'readmore' as intermediate variable:

`context["readmore"] = product.read_more`

And adding this variable manually to the context is also not necessary. You can simply use:

`{% if object.read_more %}` 

in the template. Every DetailView automatically has `object` in the context.